package-lock.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Please remove this file, we are using yarn instead of npm for installing dependencies.